src/main/java/step1/Split.java

```diff
@@ -0,0 +1,31 @@
+package step1;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Split {
+    private final static String DELIMITER = ",|:";
+    private final static String REG_DELIMITER = "//(.)\n(.*)";
+
+
+
+    public static String[] split(String inputString){
+        if(inputString == null){
+            return new String[]{"0"};
+        }
+        if(inputString.isEmpty()){
+            return new String[]{"0"};
+        }
+        return splitDelimiter(inputString);
+    }
```

Comment:
```suggestion
    public static String[] split(String inputString) {
        if (inputString == null) {
            return new String[]{"0"};
        }
        if (inputString.isEmpty()) {
            return new String[]{"0"};
        }
        return splitDelimiter(inputString);
    }
```
넥스트스텝 과정중에서는 사소하지만 이런 컨벤션도 고려해보는게 어떨까요? 😀